Add API wrapper for "Plugins" (#110)
panodata/grafana-client: docs/development.md

python -m unittest -k preference -vvv
```

### Formatting

> When Creating a PR you can run `poe format` which will help remove code style issues and keep the repo clean.

Reviewer: Just a suggestion.
```suggestion
Before creating a PR, you can run `poe format`, in order to resolve code style issues.
```
